tools/cleanlab/cleanlab_issue_handler.xml

```diff
@@ -0,0 +1,135 @@
+<tool id="cleanlab_issue_handler" name="Cleanlab Issue Handler" version="1.1.0">
+    <description>Detect and optionally clean data issues using Cleanlab</description>
+
+    <requirements>
+        <container type="docker">mojudy/galaxy_cleanlab:1.1.0</container>
+    </requirements>
+
+    <command detect_errors="exit_code">
+    <![CDATA[
+        python '${__tool_directory__}/cleanlab_issue_handler.py'
+        --csv '$input_file'
+        --task '$task'
+        --method '$method'
+        #if $summary_only:
+            --summary
+        #end if
+        #if not $label_issues:
+            --no-label-issues
+        #end if
+        #if not $outliers:
+            --no-outliers
+        #end if
+        #if not $near_duplicates:
+            --no-near-duplicates
+        #end if
+        #if not $non_iid:
+            --no-non-iid
+        #end if
+    ]]>
+    </command>
+
+    <inputs>
+        <param name="input_file" type="data" format="csv" label="Input CSV file (must contain 'target' column)"/>
```

Comment:
```suggestion
        <param name="input_file" type="data" format="csv" label="Input CSV file" help="Must contain a 'target' column)"/>
```

It would be nice if this tool could also work with tabular data.